timvt/db/catalog.py

```diff
@@ -25,7 +25,10 @@
         jsonb_object(
             array_agg(column_name),
             array_agg(udt_name)
-        ) as coldict
+        ) as coldict,
+        ST_AsText(
+            ST_Transform(ST_SetSRID(ST_EstimatedExtent(f_table_schema, f_table_name, f_geometry_column), srid), 4326)
+        ) as bounds_wkt
```

Comment:
@geospatial-jeff what's the motivation for returning this as a WKT?

Reply:
motivation was it's the first thing I thought of at the time 😆